### PR TITLE
VL-111_Document-all-expose_Vitalii-Dudnik

### PR DESCRIPTION
**Pull request by KinduD21**
https://ilevel.atlassian.net/jira/software/c/projects/VL/boards/11?selectedIssue=VL-111

### CODE REVIEW

File: src/ui.button-link/ULink.vue [hunk 1]
```diff
@@ -309,4 +307,12 @@ function onKeydown(event) {
 function onBlur(event) {
   emit("blur", event);
 }
+
+defineExpose({
+  /**
+   * A reference to the link element for direct DOM manipulation.
```
Comment (itsJohnnyGrid) [inline src/ui.button-link/ULink.vue:313]: A reference to the link wrapper element for direct DOM manipulation.

File: src/ui.form-input-money/UInputMoney.vue [hunk 2]
```diff
@@ -283,4 +281,24 @@ function onBlur() {
 function onInput(value) {
   nextTick(() => emit("input", value));
 }
+
+defineExpose({
+  /**
+   * Reference to the underlying input element inside UInput.
+   * @property {HTMLInputElement}
+   */
+  input,
```
Comment (itsJohnnyGrid) [inline src/ui.form-input-money/UInputMoney.vue:290]: inputRef ???

File: src/ui.form-input-money/UInputMoney.vue [hunk 3]
```diff
@@ -283,4 +281,24 @@ function onBlur() {
...
+
+  /**
+   * The raw, unformatted value of the input.
+   * @property {String | Number}
```
Comment (itsJohnnyGrid) [inline src/ui.form-input-money/UInputMoney.vue:294]: Does this format handled and shows well in storybook?

Comment (KinduD21) [reply src/ui.form-input-money/UInputMoney.vue:294]: image

Yes, it looks fine

File: src/ui.form-input-money/UInputMoney.vue (hunk 2 above)
Comment (itsJohnnyGrid) [reply src/ui.form-input-money/UInputMoney.vue:290]: no, I guess

**Review by itsJohnnyGrid — APPROVED** [review]

**Pull request merged**